docs/source/narr/caracteres.rst

```diff
@@ -1,22 +1,16 @@
 Codificação e caracteres especiais
 ==================================
 
-A especificação SciELO PS exige que os documentos :term:`XML` estejam codificados 
-em :term:`UTF-8`, e que declarem esta codificação na :term:`Declaração do XML`::
+A especificação :term:`SciELO PS` exige que os documentos :term:`XML` estejam codificados em :term:`UTF-8`, e que tenham indicada esta codificação na :term:`Declaração do XML`.
 
-  <?xml version="1.0" encoding="utf-8"?>
+   ``<?xml version="1.0" encoding="utf-8"?>``
 
 
-Caracteres especiais devem ser inseridos ou diretamente no documento ou por meio 
-de referências numéricas em hexadecimal. Por exemplo, o caractere sigma maiúsculo 
-deverá ser representado por ``Σ`` ou ``&#x03A3``.
+Caracteres especiais, quando utilizados, devem ser inseridos diretamente no documento ou por meio de referências numéricas em notação hexadecimal. Por exemplo, o caractere sigma maiúsculo deve ser representado por ``Σ`` ou ``&#x03A3``.
```

Comment:
@gustavofonseca e @leticiaquino : entidades deveriam aceitas somente se não for possível representar o caracter com "símbolos".
@asael-silva : se tiver que manter no formato de entidade, é obrigatório colocar o ponto-e-vírgula no final da entidade senão o XML fica mal formado.